frontend/src/components/VehicleCardFormFull.jsx

```diff
@@ -61,6 +61,7 @@ export function VehicleCardFormFull(props) {
     async function handleSubmit(e) {
         e.preventDefault()
         const techDateTime = new Date(`${techDate}T${"12:00:00"}`)
+        const additionalEquipmentSend = additionalEquipmentString.split(',')
```

Comment:
musi być osobna zmienna na to jak tylko 1 to wykorzystujesz? 
jak musi być to może jakaś inna nazwa byla by lepsza

Reply:
jak wywoływałem split na bieżąco, to sie ciekawe, rzeczy działy, więc wole zostawić jak jest